src/util/UserInfoHelperFunctions.tsx

```diff
@@ -116,14 +116,34 @@ export function canAccessTeam(hacker?: IHacker): boolean {
   );
 }
 
+export function canAccessTravel(hacker?: IHacker): boolean {
+  const status = hacker ? hacker.status : HackerStatus.HACKER_STATUS_NONE;
+
+  if (
+    status === HackerStatus.HACKER_STATUS_APPLIED ||
```

Comment:
Can't see this page if you're status: applied. Only if you're status: accepted, confirmed, checked-in